pyproject.toml

```diff
@@ -1,14 +1,21 @@
+# Note that this is pyproject file is here only for the vmtest utils.
+
 [tool.autopep8]
 max_line_length = 120
 
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
 [project]
-name = "images"
+name = "vmtest"
 version = "0.1.0"
-
 dependencies = [
-    "vmtest @ git+https://github.com/osbuild/bootc-image-builder.git",
+  "boto3==1.33.13",
+  "paramiko==4.0.0",
```

Comment:
Do we still need this dependency now since we dropped paramiko for regular ssh & scp?